back/babble/src/test/java/gg/babble/babble/restdocs/RoomApiDocumentTest.java

```diff
@@ -134,4 +141,175 @@ public void readRoomTest() throws Exception {
                     fieldWithPath("tags[].id").description("태그 Id"),
                     fieldWithPath("tags[].name").description("태그 이름"))));
     }
+
+    @DisplayName("page 번호와 태그 없이 게임에 해당하는 방 목록을 조회한다.")
+    @Test
+    public void readGameRoomsWithoutPageAndTagsTest() throws Exception {
+        // GET /api/rooms?gameId=2
+        // - 한 `page에 16개`
+        // - page는 1부터 시작한다.
+        int countOfOnePage = 16;
+
+        ResultActions actions = mockMvc.perform(get("/api/rooms?gameId=" + dummyGameId)
+            .accept(MediaType.APPLICATION_JSON_VALUE));
+
+        for (int i = 0; i < countOfOnePage; i++) {
+            actions.andExpect(status().isOk())
+                .andExpect(jsonPath("$[" + i + "].roomId").isNumber())
```

Comment:
중복 사용하는 필드를 미리 정의 해둔다.
```java
private static final String ROOM_ID = "$[%d].roomId";
private static final String CREATED_DATE = "$[%d].createdDate";
.
.
.
```

```suggestion
                .andExpect(jsonPath(ROOM_ID, i).isNumber())
                .andExpect(jsonPath(CREATED_DATE, i).isString())
                .
                .
                .
```

위와 같이 표현한다면 중복 사용하는 코드에 대한 가독성을 높일 수 있어 보이네요.

혹은 room 만 정의해두고 사용할 수도 있겠어요.
```java
private static final String ROOM = "$[%d].";
```

```
                .andExpect(jsonPath(ROOM + "roomId", i).isNumber())
                .andExpect(jsonPath(ROOM + "createdDate", i).isString())
                .
                .
                .
```

개인적인 생각이지만 지금은 가독성이 다소 떨어져 보입니다.
최소한 둘 중 취향에 맞는 것을 사용해서 리팩토링하면 좋겠습니다. 😉

Reply:
json 경로를 확인하기위해서 다시 코드 위로 올라가야된다는 단점이 있어서 고민이되네요... String format을 사용하는 점은 좋은 것같아 그 부분은 반영해 보겠습니다!